src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -427,108 +525,24 @@ export class VmNetworkTab extends React.Component<VmNetworkTabProps, VmNetworkTa
                         </Flex>
```

Comment:
Having lots of "network" types feels like a waste of this column. Could this become a "Name/Type" column and show the interface name if available? IMHO that helps an admin most to identify a device.

Reply:
Yes, I agree that this column is usually not very interesting.  There _are_ different types ("network", "direct", "bridge"), but there is more interesting stuff to show for each type.

For type "network", the name of the libvirt network object, such as "default".

For type "direct", the name of the host interface that is being attached.

For type "bridge", the name of the host bridge that is being used.

This is called the "Source" in the "Add virtual network interface" dialog, so we can do the same. Stay tuned.